inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -1507,6 +1487,52 @@ Note the possibility of negative confidence intervals if there are low counts
 ## weighted counts and proportion of dead
 death_props <- tab_survey(survey_design, died, deff = TRUE, keep = "TRUE")
 
+
```

Comment:
@amygimma - any chance you could take a look at turning the below two in to proper functions? gonna post on gtsummary to see if they are already working on this